app/views/global_error.scala.html

```diff
@@ -17,11 +17,11 @@
 @(pageTitle: String, heading: String, message: String, applicationConfig: config.ApplicationConfig = config.ApplicationConfig)(implicit request: Request[_], messages: Messages)
 
 @contentHeader = {
-<h1>@heading</h1>
+<h1 class="column-two-thirds">@heading</h1>
 }
 
 @mainContent = {
-<p>@Html(message)</p>
```

Comment:
GREY - Just a general note to verify that this isn't affecting pages incorrectly.

Reply:
Viewed page and did not see anything untoward